### DISCUSSION TITLE
Temperature unit

### DISCUSSION THREAD

**Discussion by GameTerrific** [syssi/xiaomi_fan#281] [Q&A]
Is there a way to change the temperature unit on it? I get a reading for Celsius, but it is showing F instead of C.
![asdasd](https://github.com/user-attachments/assets/a73a467d-698f-477c-8ffa-732ce7499989)
Temperature reading shown from fan. (showing degrees F and not degrees C)
![asdasds](https://github.com/user-attachments/assets/7a682ac4-456f-4604-aafa-01bd8c244fc0)
Actual temperature in my area.

**Comment by syssi** [comment]
This looks like an issue of your Lovelace card. This implementation retrieves the temperature measurement from the device but without any unit.

**Reply by GameTerrific** [reply]
My Lovelace card cannot be edited due to not having an entity id.
![asdasdadsada](https://github.com/user-attachments/assets/b05c86d8-3af9-4de8-a56c-db195962f7dd)
Would you know how to create it with an entity id?

**Reply by syssi** [reply]
If you remove the `model` parameter from your configuration.yaml section the unique ID will be generated.
